src/docker/restorableappengine.cc

```diff
@@ -554,7 +557,7 @@ void RestorableAppEngine::installAppImages(const boost::filesystem::path& app_di
     const Uri uri{Uri::parseUri(image_uri, false)};
     const std::string tag{uri.registryHostname + '/' + uri.repo + ':' + uri.digest.shortHash()};
     const auto image_dir{app_dir / "images" / uri.registryHostname / uri.repo / uri.digest.hash()};
-    installImage(client_, image_dir, blobs_root_, docker_host_, tag);
+    installImage("docker", image_dir, blobs_root_, docker_host_, tag);
```

Comment:
> It would be nice to elaborate in the commit message as to _why this is better_.

Certainly, this is just WIP, I'll provide all details in the final result.

Reply:
> So, this basically replaces `scopeo copy` with `docker pull` which uses a manifest pointing at locally pre-downloaded files, right?

Almost, not `docker pull` but `docker load`. I actually think we don't even need to invoke the CLI command, most likely simple `HTTP POST /image/load` will work.